files/mods.json

```
@@ -642,7 +641,7 @@
         "creator": "Biscuit",
         "discordcode": "qBkzcgdPqU",
         "command": "/sba",
-        "description": "Essential QOL Skyblock mod with a plethora of indispensable features, locking slots, timers, and a ton more to discover.",
+        "description": "Skyblock mod with many features, like locking slots, customizing your HUD, and more",
```

```suggestion
        "description": "Skyblock mod with many features, like customizing your HUD, a custom warp menu, and more",
```
neu's slot locking is better